odoo/addons/base/tests/test_ir_model.py

```diff
@@ -155,3 +158,13 @@ def test_create_xmlids(self):
 
         self.assertEqual(self.get_data('test_convert.foo').noupdate, True)
         self.assertEqual(self.get_data('test_convert.bar').noupdate, True)
+
+    @mute_logger('odoo.sql_db', 'odoo.addons.base.models.ir_model')
+    def test_create_external_id_with_space(self):
+        model = self.env['res.partner.category']
+        data_list = [
+            dict(xml_id='test_convert.with space', values={'name': 'Bar'}),
+        ]
+        name_check = 'ir_model_data_name_nospaces'
```

Comment:
Not sure the local name brings much to the table, just inline it.

Reply:
Done.